packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -120,6 +126,9 @@ export class StripeHelper {
         statsd.timing('stripe_request', response.elapsed);
       });
     }
+
+    // Establish database connection and bind instance to Model using Knex
+    setupAuthDatabase(config.database.mysql.auth);
```

Comment:
Setup tasks like this should be in `bin/key_server.ts` where other root objects are instantiated. The Redis instance should've also been setup there, and then fetched using `typedi`, but we didn't have that in place back then. We actually have the `statsd` object registered with a container now, so passing it all the way down the chain isn't needed either, this could prolly use some cleanup in general.